optlang/symbolics.py

```diff
@@ -84,12 +86,14 @@ def __getnewargs__(self):
     def add(*args):
         if len(args) == 1:
             args = args[0]
-        return sum(args)
+        return Add(*args)
 
     def mul(*args):
         if len(args) == 1:
             args = args[0]
-        return reduce(operator.mul, args, 1)
+        if len(args) == 0:
```

Comment:
What is the use case for calling `mul` without arguments? Also I would argue that `mul()` should return One, the identity element for multiplication.

Reply:
Yeah I just wanted to always return an expression object. What does sympy return in that case (should be consistent).